src/ivc/sangria/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -898,7 +898,12 @@ where
 
         let assigned_step_circuit_instances =
             match self.relaxed.step_circuit_instances_hash_accumulator {
-                SCInstancesHashAcc::None => SCInstancesHashAcc::None,
+                SCInstancesHashAcc::None => {
```

Comment:
if we don't absorb both off-circuit and on-circuit. would it still be okay?

Reply:
We can do that too, it's just that the code is a little harder to debug and the overhead is low.